From 7ce6fabd78aef8f6e3416fe95eee68a9b96b1e41 Mon Sep 17 00:00:00 2001
From: Antonio Terceiro <antonio.terceiro@linaro.org>
Date: Thu, 3 Feb 2022 14:01:42 -0300
Subject: [PATCH] lava_server.settings.dev: prepend, not replace, source dir to
 DEVICE_TYPES_PATHS

The source directory is listed first, so it still gets precedence over
the system-wide paths. This allows to run the tests against the
installed code (e.g. in autopkgtest).
---
 lava_server/settings/dev.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lava_server/settings/dev.py b/lava_server/settings/dev.py
index 59e5740cc..7be7b52ae 100644
--- a/lava_server/settings/dev.py
+++ b/lava_server/settings/dev.py
@@ -69,7 +69,9 @@ STATIC_ROOT = str(PROJECT_STATE_DIR / "static")
 DISPATCHER_CONFIG_PATH = str(PROJECT_SRC_DIR / "etc/lava-server/dispatcher.d")
 # Use device configuration files from source tree
 DEVICES_PATH = str(PROJECT_SRC_DIR / "etc/dispatcher-config/devices")
-DEVICE_TYPES_PATHS = [str(PROJECT_SRC_DIR / "etc/dispatcher-config/device-types")]
+DEVICE_TYPES_PATHS = [
+    str(PROJECT_SRC_DIR / "etc/dispatcher-config/device-types")
+] + DEVICE_TYPES_PATHS
 HEALTH_CHECKS_PATH = str(PROJECT_SRC_DIR / "etc/dispatcher-config/health-checks")
 
 # Make this unique, and don't share it with anybody.
-- 
2.34.1

